### PR TITLE
Power Cells and Power Cell Press

### PR DESCRIPTION
**Pull request by Xeni91**
* added textures for press and cells
* added xml for press and cells
* rendering of press does not work properly don't know why
* Press job is at the moment just a dummy, have to find out why the job breaks the game

Edit: added better markdown thanks to @Mizipzor

### CODE REVIEW

**Comment by Xeni91** [comment]
just resolved the merge conflict

**Comment by Xeni91** [comment]
Discussion about this is in #304

**Comment by Mizipzor** [comment]
Here's a friendly tip if you're new to markdown, if you leave a space between the asterisk and the first character Github would render it as an actual list. So instead of

*foo
*bar

You get

* foo
* bar

**Comment by Xeni91** [comment]
Finished evrything I wanted I am just clueless why the Power Cell sprite is not loaded the way it should be

**Comment by vogonistic** [comment]
@Xeni91 Can you remove the change to `JobSpriteController.cs` and fix the indentation in `Furniture.xml`?

File: Assets/StreamingAssets/LUA/Furniture.lua [hunk 1]
```diff
@@ -283,6 +283,68 @@ function MetalSmelter_JobComplete(j)
...
+			Job.JobPriority.Medium,
+			false
+			)
+			
+			j.RegisterJobCompletedCallback("PowerCellPress_JobComplete")
+			
+			furniture.AddJob(j)
+		end
+	else
+		furniture.ChangeParameter("smelttime", deltaTime)
```
Comment (Ceddini) [inline Assets/StreamingAssets/LUA/Furniture.lua]: @Xeni91 I think you have a small mistake here.
In the Furniture.xml you defined the Params as 
    <Param name="presstime" value="0" />
So it should be presstime not smelttime, or am I reading it wrong?

**Comment by Mizipzor** [comment]
At least for .cs files we use spaces instead of tabs. But as far as I know we havent explicitly said anything about .lua and .xml. But maybe we should. I just assumed spaces were used there as well.

**Comment by Xeni91** [comment]
Thanks for the feedback I will get to it shortly ... well sorry for indents but @ work we use them so yeah ^^

**Comment by Mizipzor** [comment]
Just wanted to point out that tabs and spaces were already mixed in that file, I'm not blaming you specifically. :)

**Comment by Xeni91** [comment]
Yeah you are right ... sometimes there are only 2 spaces for indention and sometimes there are 4, but I used Tab instead of spaces that was zu problem

**Comment by wilcomega** [comment]
I made another way of generating power: #466 
someone will have to decide which one will be the final one

**Comment by Grenkin1988** [comment]
#506 Made changes in order to enable power system to react on changes to ```powerValue```

File: Assets/Scripts/Controllers/JobSpriteController.cs [hunk 2]
```diff
@@ -22,13 +22,10 @@ public class JobSpriteController : MonoBehaviour
...
-        jobGameObjectMap = new Dictionary<Job, GameObject>();
-        fsc = GameObject.FindObjectOfType<FurnitureSpriteController>();
-<<<<<<< HEAD
+		jobGameObjectMap = new Dictionary<Job, GameObject>();
+		fsc = GameObject.FindObjectOfType<FurnitureSpriteController>();
 
-=======
->>>>>>> FuelingSystem
-        WorldController.Instance.world.jobQueue.cbJobCreated += OnJobCreated;
+		WorldController.Instance.world.jobQueue.cbJobCreated += OnJobCreated;
```
Comment (Mizipzor) [inline Assets/Scripts/Controllers/JobSpriteController.cs]: In this file it seems you went the other way around, spaces became tabs. :)

I think your Monodevelop isnt configured correctly. Check the contribution guidelines, I think there are instructions in there.

File: Assets/Scripts/Controllers/JobSpriteController.cs [hunk 3]
```diff
@@ -22,10 +22,10 @@ public class JobSpriteController : MonoBehaviour
     // Use this for initialization
     void Start()
     {
-        jobGameObjectMap = new Dictionary<Job, GameObject>();
-        fsc = GameObject.FindObjectOfType<FurnitureSpriteController>();
-        
-        WorldController.Instance.world.jobQueue.cbJobCreated += OnJobCreated;
+		jobGameObjectMap = new Dictionary<Job, GameObject>();
+		fsc = GameObject.FindObjectOfType<FurnitureSpriteController>();
```
Comment (zwrawr) [inline Assets/Scripts/Controllers/JobSpriteController.cs]: Tabs wanna be changed

**Comment by vogonistic** [comment]
Merged!

**Comment by ComputerOverlord** [comment]
@vogonistic Why did you merge this AND uranium?

**Comment by vogonistic** [comment]
To get the resources in the game. You can open a new PR to make power cells out of Uranium.

**Pull request merged**